packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -1796,5 +1797,5 @@ void main() {
         affinity: TextAffinity.upstream,
       ));
     }, variant: macOSOnly);
-  });
+  }, skip: kIsWeb); // [intended]
```

Comment:
this needs a comment saying why it is ok to skip these on web. (see other examples of this in the repository)

Reply:
Updated this one and other comments on "skip: kIsWeb" in the file.